pack/tsk/tskp.json

```diff
@@ -1,81 +1,542 @@
+    },
+    {
+        "code": "09031",
+        "deck_limit": 2,
+        "faction_code": "guardian",
+        "illustrator": "Rob Laskey",
+        "name": "Helping Hand",
+        "pack_code": "tskp",
+        "position": 31,
+        "quantity": 2,
+        "text": "You may commit Helping Hand to any type of test. Max 1 committed per test.\nWhile Helping Hand is committed to a skill test, double the skill icons of each other card committed to that test.",
+        "traits": "Innate.",
+        "type_code": "skill",
+        "xp": 0
+    },
+    {
+        "code": "09033",
+        "cost": 4,
+        "deck_limit": 2,
+        "faction_code": "guardian",
+        "flavor": "Day 15. The entities appear to be feeding. On what, I cannot imagine. I dare not approach further.",
+        "health": 2,
+        "illustrator": "Sedone Thongvilay",
+        "name": "Field Agent",
+        "pack_code": "tskp",
+        "position": 33,
+        "quantity": 2,
+        "sanity": 2,
+        "skill_agility": 1,
+        "skill_intellect": 1,
+        "slot": "Ally",
+        "text": "You get +1 [intellect].\n[fast] Exhaust Field Agent and deal 1 horror to it: Discover 1 clue at your location.",
+        "traits": "Ally. Agency.",
+        "type_code": "asset",
+        "xp": 2
+    },
+    {
+        "code": "09047",
+        "cost": 1,
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "flavor": "\"Don't you realize what this means?\"\n\"Why don't you calm down and tell me very slowly?\"",
+        "illustrator": "Stephen Somers",
+        "name": "Captivating Discovery",
+        "pack_code": "tskp",
+        "position": 47,
+        "quantity": 2,
+        "skill_agility": 1,
+        "skill_intellect": 1,
+        "text": "Search the top 6 cards of your deck. You may place up to 3 of your clues on your location. For each clue placed in this way, choose and add 2 of the searched cards to your hand. Shuffle your deck.",
+        "traits": "Insight.",
+        "type_code": "event",
+        "xp": 0
+    },
+    {
+        "code": "09067",
+        "cost": 0,
+        "deck_limit": 2,
+        "faction_code": "rogue",
+        "illustrator": "Aleksander Karcz",
+        "name": "\"I'll take that!\"",
+        "pack_code": "tskp",
+        "position": 67,
+        "quantity": 2,
+        "skill_agility": 1,
+        "skill_intellect": 1,
+        "text": "Fast. Play when you successfully evade a [[Humanoid]] enemy or successfully investigate.\nPlay an [[Item]] asset from your hand, reducing its cost by X, where X is the amount you succeeded by. Attach \"I'll take that!\" to that asset. Attached asset gains [[Illicit]].",
+        "traits": "Trick. Upgrade. Illicit.",
+        "type_code": "event",
+        "xp": 0
+    },
+    {
+        "code": "09080",
+        "cost": 2,
+        "customization_options": [
+            {
+                "xp": 1,
+                "health": 3
+            },
+            {
+                "xp": 1
```

Comment:
Yeah, the rule I've been doing is 'does it impact deck building' -- since none of the text inside the actual choices gets translated, and it seems like its going to be impossible to actually composite the text into the 'final form' of the card without a LOT more work.